decidim-core/lib/decidim/api/types/static_page_type.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    # This type represents a page
+    class StaticPageType < Decidim::Api::Types::BaseObject
+      description "The current organization static pages"
+
+      implements Decidim::Core::TimestampsInterface
+
+      field :content, Decidim::Core::TranslatedFieldType, "The content of this page.", null: true
+      field :id, GraphQL::Types::ID, "The id of the page", null: false
+      field :title, Decidim::Core::TranslatedFieldType, "The title of this page.", null: false
+      field :topic, Decidim::Core::StaticPageTopicType, "The topic of this page", null: true
```

Comment:
Inconsistency in the usage of the dots for finishing the sentence. (Is not something particular of this PR, but let's try to be consistent at least in the file level)

```suggestion
      field :content, Decidim::Core::TranslatedFieldType, "The content of this page", null: true
      field :id, GraphQL::Types::ID, "The id of the page", null: false
      field :title, Decidim::Core::TranslatedFieldType, "The title of this page", null: false
      field :topic, Decidim::Core::StaticPageTopicType, "The topic of this page", null: true
```